core/src/nucleus/actions/call_zome_function.rs

```diff
@@ -15,6 +15,7 @@ use holochain_core_types::{
     },
     error::HolochainError,
     signature::{Provenance, Signature},
+    ugly::lax_send_sync,
```

Comment:
I think we decided that is no longer "ugly", i.e. it's actually a pretty reasonable thing to do in our case. I could move this to a less derogatory module.